ReactWindows/ReactNative.Shared/UIManager/Events/EventDispatcher.cs

```diff
@@ -296,13 +296,13 @@ private void ScheduleDispatch(object sender, object e)
                 _reactContext.RunOnJavaScriptQueueThread(() =>
                 {
                     DispatchEvents(activity);
-                    activity.Dispose();
+                    activity?.Dispose();
                 });
 
                 return;
             }
 
-            activity.Dispose();
+            activity?.Dispose();
```

Comment:
Was doing `using (activity) { }` before because it's effectively:
```
try { } finally { if (activity != null) activity.Dispose() }
```
However, there's no real reason to do this in a finally anyway.